requirements.txt

```
@@ -14,6 +14,7 @@ docutils==0.17.1
 execnet==1.9.0
 filelock==3.3.1
 ghp-import==2.1.0
+httpx==0.27.0
```

```suggestion
httpx==0.28.1
```
https://www.python-httpx.org